src/main/java/net/minestom/server/command/CommandSender.java

```diff
@@ -1,6 +1,9 @@
 package net.minestom.server.command;
 
+import java.util.UUID;
```

Comment:
Unused import?

Reply:
Appears to be, I'll update this PR tomorrow at some point.